scene/gui/view_panner.h

```diff
@@ -63,6 +64,8 @@ class ViewPanner : public RefCounted {
 	bool enable_rmb = false;
 	bool simple_panning_enabled = false;
 
+	Viewport *viewport;
```

Comment:
```suggestion
	Viewport *viewport = nullptr;
```

Reply:
Thanks for your comments. I have included all your changes.
For future reference, should I generally follow the rule to initialize all pointers with `nullptr`?